env.base

```diff
@@ -264,7 +264,8 @@ ENABLE_COOG_TOKEN_AUTHENTICATION=${ENABLE_COOG_TOKEN_AUTHENTICATION:-true}
 JWT_ENCRYPTION=${JWT_ENCRYPTION:?}
 JWT_INTERNAL_ENCRYPTION=${JWT_ENCRYPTION}
 JWT_EXPIRATION=${CUSTOM_JWT_EXPIRATION:-3600}
-DOCUMENT_TOKEN_SECRET=${CUSTOM_DOCUMENT_TOKEN_SECRET-$JWT_ENCRYPTION}
+DOCUMENT_TOKEN_SECRET=${CUSTOM_DOCUMENT_TOKEN_SECRET-${JWT_ENCRYPTION}}
+PASSWORD_TOKEN_SECRET=${CUSTOM_PASSWORD_TOKEN_SECRET-${JWT_ENCRYPTION}}
```

Comment:
je l'appelerai bien TOKEN_SHARED_SECRET , pour indique que c'est partagé entre le front et le back . ou un truc du genre. tu en penses quoi ? 

--

in any case, ça me fait penser qu'il faut aussi ajouter TRYTOND_B2C_API__SHARED_SECRET et y mettre la valeur, dans le common de coog . tu veux que je le fasse ?

Reply:
@MaximeCoopengo ?

Reply:
Pas de soucis pour changer le nom de la variable d'env, Fabien m'avait indiqué celle là, mais on peut utiliser celle que vous voulez.
Ok pour changer avec `TOKEN_SHARED_SECRET`.

Je peux rajouter `TRYTOND_B2C_API__SHARED_SECRET`, j'utilise `JWT_ENCRYPTION` en valeur par défaut également ?

Reply:
ok , si tu peux le faire c'est top ! merci @MaximeCoopengo